contracts/ContinuousSale.sol

```diff
@@ -1,38 +1,64 @@
 pragma solidity ^0.4.11;
 
-import "zeppelin-solidity/contracts/crowdsale/Crowdsale.sol";
+import "zeppelin-solidity/contracts/token/MintableToken.sol";
 
 /**
- * @title ContinuousCrowdsale
- * @dev ContinuousCrowdsale implements a contract for managing a continuous token sale
+ * @title ContinuousSale
+ * @dev ContinuousSale implements a contract for managing a continuous token sale
  */
-contract ContinuousCrowdsale is Crowdsale {
+contract ContinuousSale {
+    using SafeMath for uint256;
+
     // time bucket size
     uint256 public constant BUCKET_SIZE = 12 hours;
 
-    // true if continous sale is enabled
-    bool public continuousSale = false;
+    // the token being sold
+    MintableToken public token;
+
+    // address where funds are collected
+    address public wallet;
+
+    // amount of tokens emitted per wei
+    uint256 public rate;
+
+    // amount of raised money in wei
+    uint256 public weiRaised;
+
+    // max amount of tokens to mint per time bucket
+    uint256 public issuance;
```

Comment:
The `issuance` here is computed from the token supply when the contract receives the `start` call, and is fixed after that.

What do you think would be the best strategy here? We could do an explicit initialization in the constructor for the purpose of cleanness.